test/test_helper.rb

```diff
@@ -2,3 +2,79 @@
 require 'tyrant'
 
 require 'minitest/autorun'
+
+require 'warden'
+require 'rack-minitest/test'
+
+# Use Warden spec helper to simulate Warden
+# https://github.com/hassox/warden/blob/master/spec/helpers/request_helper.rb
+module Warden::Spec
```

Comment:
Yeah, we have to ask them to make that helper available via `lib/test/helpers.rb`, we do the same in Cells! Do you want to PR them?